containers/nginx-storage/default.conf

```diff
@@ -0,0 +1,12 @@
+error_log logs/error.log notice;
+
+server {
+    # HTTP
+    listen 81       default_server;
+
+    # NOTE: OpenResty uses /usr/local/openresty/nginx/, not /etc/nginx/ as the default config location.
+    # We have to use absolute path to include files from /etc/nginx/.
+    include                  /etc/nginx/snippets/error-pages.conf;
```

Comment:
Please align this properly.

Reply:
Why is not correct now ? This is not change in this pull request.

Reply:
Well, if you did not touch this line in this commit (only copied it) then it can stay. But please refactor it later.

This should look like this:
```
include /etc/nginx/snippets/error-pages.conf;
```

Reply:
Done.